templates/solutions/telco/index.html

```diff
@@ -1,30 +1,39 @@
 {% extends 'base_index.html' %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1yRSYYY8_UNfZzdLcp_fYxi-STGm40o3UP9vD2vW6brw{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1yRSYYY8_UNfZzdLcp_fYxi-STGm40o3UP9vD2vW6brw
+{% endblock meta_copydoc %}
 
 {% block title %}Canonical Telco Solutions{% endblock %}
 
-{% block meta_description %}Telco projects are more cost-effective and innovative with open source. Canonical helps you navigate and operate the complex landscape of telco software with confidence and at scale.{% endblock %}
+{% block meta_description %}
+  Telco projects are more cost-effective and innovative with open source. Canonical helps you navigate and operate the complex landscape of telco software with confidence and at scale.
+{% endblock %}
 
-{% block body_class %}is-paper{% endblock body_class %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
 
-<section class="p-strip is-shallow u-no-padding--bottom">
-  <div class="p-section">
-    <div class="row--50-50 p-section">
-      <div class="col">
-        <h1>Transform your infrastructure with secure and cloud-native telecom solutions</h1>
-      </div>
-      <div class="col">
-        <p>Canonical is the trusted and agile partner you need to boost network performance, optimise costs and achieve digital transformation with open source.</p>
-        <p>Get enterprise-grade security and support to innovate with confidence.</p>
-        <hr />
-        <a class="p-button--positive" href="https://ubuntu.com/telco#get-in-touch">Contact us</a>
+  <section class="p-section--shallow">
```

Comment:
```suggestion
  <section class="p-strip--shallow">
```
There is a little padding on top that is missing